backend/django/scripts/isort.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+#
+# Helper script for running isort formatter
+# By default this script will run in --check mode
+#   if --format is passed then script will format code
+
+set -euo pipefail
+IFS=$'\n\t'
+
+if [[ $* == --format ]]; then
+  isort -rc /code/
```

Comment:
https://pycqa.github.io/isort/docs/upgrade_guides/5.0.0.html for version 5, the -rc flag is no longer used as isort is now recursive by default

Reply:
done